Ajout de Blockusign dans Signature en ligne

KeziahMoselle opened the pull request.

Commit:
Ajout : Blockusign dans Signature en ligne

Commit:
Sort

etnbrd commented:
Hey, merci. 2ème contribution en une semaine, c'est très cool de ta part :)

Juste un petit point, je suis assez partant pour mettre une dapp dans la liste (même en beta), mais est-ce qu'on peut dire qu'elle est gratuite ? On paye avec de l'ether, ou du gas, non ?

KeziahMoselle commented:
C'est une très bonne question, je n'ai pas vu ces informations ... Alors j'ai mis gratuit par défaut vu que je n'ai pas trouvé de pricing table ou autre :/

etnbrd commented:
Bon, on va pas mettre free, et passer ça comme ça :)

etnbrd committed:
Update readme.md

Remove free from Blockusign, as it is backed by ethereum blockchain, hence not free per se.
